recipes/pycld2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pycld2" %}
+{% set version = "0.41" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+    - wheel >=0.23.0
+    - libstdcxx-ng
+    - libgcc-ng
```

Comment:
```suggestion
```

Reply:
I think it was complaining about not finding libstdc++ (or something) before those were added.

Reply:
Build linux fail with this message : 
`but ['libstdcxx-ng'] not in reqs/run, [...] ['libgcc-ng'] not in reqs/run`